<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLogin</class>
 <widget class="QDialog" name="CLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pPB1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="digit" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pPB2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="digit" stdset="0">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pPB3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="digit" stdset="0">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pPB4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="digit" stdset="0">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pPB5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="digit" stdset="0">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pPB6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="digit" stdset="0">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pPB7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="digit" stdset="0">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pPB8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="digit" stdset="0">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pPB9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="digit" stdset="0">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pPB0">
       <property name="text">
        <string>0</string>
       </property>
       <property name="digit" stdset="0">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="pWidgetControls" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pPBClear">
        <property name="text">
         <string>Clear entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pPBSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pLInfo">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pPBOk">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pPB1</tabstop>
  <tabstop>pPB2</tabstop>
  <tabstop>pPB3</tabstop>
  <tabstop>pPB4</tabstop>
  <tabstop>pPB5</tabstop>
  <tabstop>pPB6</tabstop>
  <tabstop>pPB7</tabstop>
  <tabstop>pPB8</tabstop>
  <tabstop>pPB9</tabstop>
  <tabstop>pPB0</tabstop>
  <tabstop>pPBClear</tabstop>
  <tabstop>pPBSave</tabstop>
  <tabstop>pPBCancel</tabstop>
  <tabstop>pPBOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
